packages/formation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/formation",
-  "version": "7.1.0-color-beta",
+  "version": "8.0.0",
```

Comment:
Bumped major version since removing could be a breaking change. Let me know if you disagree